deps/rabbit/src/rabbit_quorum_queue.erl

```diff
@@ -406,7 +406,14 @@ handle_tick(QName,
                                {messages_persistent, M}
 
                                | infos(QName, ?STATISTICS_KEYS -- [consumers])],
-                      rabbit_core_metrics:queue_stats(QName, Infos),
+                      {ok, Queue} = rabbit_amqqueue:lookup(QName),
+                      Up = case ra:consistent_query(amqqueue:get_pid(Queue), fun (_) -> ok end) of
+                          {ok, ok, _} -> 1;
+                          _ ->
+                              rabbit_log:debug("ra:consistent_query failed for queue ~p~n", [QName]),
```

Comment:
Instead of using `~p` on queue name, consider `rabbit_misc:rs/1`

Reply:
Done.